laguinho/models/metadata.py

```diff
@@ -0,0 +1,9 @@
+from marshmallow import Schema, fields, pprint, validate
+
+class MetadataSchema(Schema):
+    name = fields.Str(required=True, error_messages={"required": {"message": "Name is required."}})
+    url = fields.Str(required=True, error_messages={"required": {"message": "URL is required."}})
+    path = fields.Str(required=True, error_messages={"required": {"message": "Path is required."}})
+    maintainers = fields.List(fields.Str())
+    format = fields.Str(validate=validate.OneOf(["json", "csv"]), required=True, error_messages={"required": {"message": "Format is required."}})
```

Comment:
Acho que seria legal a gente começar desde já a utilizar os padrões do [PEP8](https://www.python.org/dev/peps/pep-0008/) pra documentação e codificação mesmo. Até facilita a vida de quem for começar a contribuir.